### ISSUE TITLE
Missing translation domain

### ISSUE THREAD

**Issue by xfxjzhflct** [simplesamlphp/simplesamlphp-module-consent#28]
Hello everybody,
since updating simplSAMLphp to v2.3.4, we've noticed that some messages in the consent UI are no longer being translated.

The following screenshot should illustrate the issue 
![Screenshot_20241202_091459](https://github.com/user-attachments/assets/1f307385-b2a6-4f81-9976-872f2648a2eb)

It appears that the translation files do not have a translation domain set. 

If we add the line

`"X-Domain: messages\n"`

to the preamble of the file locales/de/LC_MESSAGES/consent.po, translations are resolved correctly again. The same happens for languages other than German as well.

Thanks for your help

**Comment by tvdijen** [comment]
Yes, we have been made aware of this regression. We will tag a new release somewhere during the day.

**Comment by xfxjzhflct** [comment]
No need to apologize. Thanks again for your quick work.
